dali/pipeline/operator/builtin/conditional/scope_argument.cc

```diff
@@ -0,0 +1,21 @@
+// Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operator/operator.h"
+
+DALI_SCHEMA(ImplicitScopeAttr)
+    .AddHiddenOptionalArg<int>("_scope", R"code(Any CPU batch to infer the current batch size from.
```

Comment:
```suggestion
    .AddHiddenOptionalArg<int>("_scope", R"code(Any batch to infer the current batch size from.
```
That's just current limitation that we don't need to repeat here and may lift in the future.

Reply:
done